apps/spark/src/main/java/com/linkedin/openhouse/jobs/spark/TableStatsCollectionSparkApp.java

```diff
@@ -0,0 +1,39 @@
+package com.linkedin.openhouse.jobs.spark;
+
+import com.linkedin.openhouse.jobs.spark.state.StateManager;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Option;
+
+/**
+ * Class with main entry point to collect Iceberg stats for a table.
+ *
+ * <p>Example of invocation: com.linkedin.openhouse.jobs.spark.TableStatsCollectionSparkApp
+ * --tableName db.testTable
+ */
+@Slf4j
+public class TableStatsCollectionSparkApp extends BaseTableSparkApp {
+
+  public TableStatsCollectionSparkApp(String jobId, StateManager stateManager, String fqtn) {
+    super(jobId, stateManager, fqtn);
+  }
+
+  @Override
+  protected void runInner(Operations ops) {
+    log.info("Running TableStatsCollectorApp for table {}", fqtn);
+
+    ops.collectTableStats(fqtn);
+  }
+
+  public static void main(String[] args) {
+    List<Option> extraOptions = new ArrayList<>();
+    extraOptions.add(new Option("t", "tableName", true, "Fully-qualified table name"));
```

Comment:
Are we using tableName, or fqtn as a standard in other job types? Let's make sure we are consistent.

Reply:
This usage is consistent with other jobs that we have today. https://github.com/linkedin/openhouse/blob/main/apps/spark/src/main/java/com/linkedin/openhouse/jobs/spark/SnapshotsExpirationSparkApp.java#L83